folioreader/src/main/java/com/folioreader/activity/ContentHighlightActivity.java

```diff
@@ -13,22 +13,21 @@
 import com.folioreader.R;
 import com.folioreader.fragments.ContentsFragment;
 import com.folioreader.fragments.HighlightListFragment;
-import com.folioreader.util.AppUtil;
 import com.folioreader.util.UiUtil;
 
-import nl.siegmann.epublib.domain.Book;
-
 public class ContentHighlightActivity extends AppCompatActivity {
-    private Book mBook;
+    private String mBookTitle;
     private int mSelectedChapterPosition;
     private boolean mIsNightMode;
+    private String mBookPath;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_content_highlight);
         getSupportActionBar().hide();

```

Comment:
Hey, could you guys add a fix here? If I use a theme with no action bar in my app theme, this will crash because getSupportActionBar() will return a null value. Just check if it isn't null then hide. In the current state, I had to add an exclusive line of code to set a different style to this Activity.

```
ActionBar actionBar = getSupportedActionBar();
if (actionBar != null)
    actionBar.hide();
```